eZ/Publish/SPI/Compare/CompareField.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace eZ\Publish\SPI\Compare;
+
+use eZ\Publish\SPI\Persistence\ValueObject;
+
+abstract class CompareField extends ValueObject
+{
+    /** @var string */
+    public $name;
+
+    /** @var array|int|string */
```

Comment:
```suggestion
    /** @var mixed */
```
I assume that other types are allowed, e.x. `float`